documentation/modules/ROOT/pages/transformations/partition-routing.adoc

```diff
@@ -0,0 +1,140 @@
+:page-aliases: configuration/partition-routing.adoc
+// Category: debezium-using
+// Type: assembly
+// ModuleID: route-record-to-specific-partition-base-on-specific-table-column
+// Title: Partition Routing
+[id="partition-routing"]
+= Partition Routing
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+ifdef::community[]
+[NOTE]
+====
+This single message transformation (SMT) is supported for only the MySQL database connector.
+====
+endif::community[]
+
+As described in xref:install.adoc#configuring-debezium-topics[Installing Debezium], by default, table changes will be sent to a topic with a single partition.
+If you decide to change this configuration as described in xref:configuration/topic-auto-create-config.adoc[Customization of Kafka Connect automatic topic creation], Kafka will determine the partition by hashing the key by default.
+
+You can use this SMT if you want to use a specific table column to calculate the destination partition. The hash of the value will be used to compute the destination partition.
+
+// Type: concept
+// Title: Example: {prodname} basic partition routing SMT configuration
+// ModuleID: example-basic-debezium-partition-routing-smt-configuration
+[[example-basic-partition-routing-configuration-example]]
+== Example: Basic configuration
+
+You configure the partition routing transformation in the {prodname} connector's Kafka Connect configuration.
+In the configuration, you specify the tables that you are interested in, the table column you want to use to calculate the partition and the max number of partition configured for the tables.
+Only events that come from configured tables will be processed. The others will be simply leaved as is.
+
+To configure a {prodname} connector to route events to a specific partition, configure the `ComputePartition` SMT in the Kafka Connect configuration for the {prodname} connector.
+
+For example, you might add the following configuration in your connector configuration.
+
+[source]
+----
+...
+topic.creation.default.partitions=2
+topic.creation.default.replication.factor=1
+...
+transforms=ComputePartition
+transforms.ComputePartition.type=io.debezium.connector.mysql.transforms.partitions.ComputePartition
+transforms.ComputePartition.partition.table.list=products,orders
```

Comment:
Use qualified table names in examples.

Reply:
So you would like to have  `inventory.products?` but why only on example? I can for sure put it in the example but then I need to correctly extract the database and table name from this to correctly match the `db` and `name` filed in the `source`

Reply:
@mfvitale Yes, you are right, please see other comments. It is necessary to take both db/schema name into consideration otherwise you could not handle same  table names in different schemas.